### PR TITLE
[DPC-4246] Disable default logging in opt-out-import lambda

### PR DESCRIPTION
**Pull request by ashley-weaver**
## 🎫 Ticket

https://jira.cms.gov/browse/DPC-4246

## 🛠 Changes

Removed default logging from AWS SDK Golang config.

## ℹ️ Context

The AWS SDK (Golang, v2) automatically logs errors to stderr in a non-JSON format that is not parseable by Splunk. This is not necessary since our application itself will catch and log errors in the standardized format, with the standardized log information.

### CODE REVIEW

File: lambda/opt-out-import/main.go [hunk 1]
```diff
@@ -194,9 +195,13 @@ func importResponseFile(bucket string, file string) (int, int, string, error) {
 }
 
 func createV2Cfg() (*awsv2.Config, error) {
-	cfg, err := config.LoadDefaultConfig(context.TODO(), config.WithRegion("us-east-1"))
-	assumeRoleArn, err := getAssumeRoleArn()
+	// disable default logging from aws-sdk-go-v2
+	cfg, err := config.LoadDefaultConfig(context.TODO(), config.WithLogger(logging.Nop{}))
```
Comment (jdettmannnava) [inline lambda/opt-out-import/main.go:199]: Do we not need region?

**Comment by jdettmannnava** [comment]
Any way to validate?

File: lambda/opt-out-import/main.go (hunk 1 above)
Comment (ashley-weaver) [reply lambda/opt-out-import/main.go:199]: this is to remove the default logging entirely, so region is unnecessary

Comment (jdettmannnava) [reply lambda/opt-out-import/main.go:199]: Is this not the config for everything, not just logging?

Comment (ashley-weaver) [reply lambda/opt-out-import/main.go:199]: this is how the docs say to configure logging
https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/logging/
https://pkg.go.dev/github.com/aws/smithy-go/logging

Comment (jdettmannnava) [reply lambda/opt-out-import/main.go:199]: Sure, but I think you can leave the region in, as well?

**Review by jdettmannnava — APPROVED** [review]
LGTM

**Pull request merged**